modeler/cayenne-modeler/src/main/java/org/apache/cayenne/modeler/dialog/db/load/DbImportTreeNode.java

```diff
@@ -207,4 +210,45 @@ public boolean isLoaded() {
     public void setLoaded(boolean loaded) {
         isLoaded = loaded;
     }
+
+    public List<DbImportTreeNode> getChildNodes() {
+        List<DbImportTreeNode> childNodes = new ArrayList<>();
```

Comment:
Could it be just `return Collections.list(...)`?

Reply:
We need in casting in DbTreeNode here, so we can't just to collect Enumeration in List. I propose something like this 

`return this.children.stream().map(c -> (DbImportTreeNode) c).collect(Collectors.toList());`